### PR TITLE
Add mnemonic-indexes parameter to README

### PR DESCRIPTION
**Pull request by Rubilmax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +36,9 @@ With `make sign` & `make exec`, one can also use any other wallet provider avail
 - `make cmd:keystore` to use a keystore
 - `make cmd:"private-key 0x..."` if you really want to save your private key to your shell's history...
 
+You can also append any `cast` parameter:
+- `make cmd:"ledger --mnemonic-indexes 1"` to use another account than the default one at index `0`
```
Comment (julien-devatom) [inline README.md:40]:
```suggestion
- `make cmd:"ledger --mnemonic-index 1"` to use another account than the default one at index `0`
```

**Review by julien-devatom — CHANGES_REQUESTED** [review]
Is there a w

File: README.md (hunk 1 above)
Comment (Rubilmax) [reply README.md:40]: The foundry book seems to indicate it is plural: https://book.getfoundry.sh/reference/cast/cast-wallet-address

**Comment by Rubilmax** [comment]
> Is there a way to retrieve the address used before signing?

There could be, using `cast wallet address`, but it would require to interact twice with the HW or console, which is cumbersome. The only way would be to have it implemented in `cast` directly

**Review by MerlinEgalite — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (julien-devatom) [reply README.md:40]: Ok after having tested w the latest forge version (forge 0.2.0 (31fcf5a 2023-05-19T00:04:19.980323193Z)), using plural for `cast wallet sign` gave me the following error:
image

But, for exec, only plural works
```bash
 make exec:"ledger --mnemonics 1  --mnemonic-indexes 1"
```

Comment (julien-devatom) [reply README.md:40]: I think you can document the 2 workflows (sign & exec) with the current working flow

**Review by MerlinEgalite — APPROVED** [review]

**Review by julien-devatom — APPROVED** [review]

**Pull request merged**